src/test/scala/com/campudus/tableaux/LinkTest.scala

```diff
@@ -224,7 +381,7 @@ class LinkTest extends TableauxTestBase {
     )
 
     for {
-      // setup two tables
+    // setup two tables
```

Comment:
Strange intendation? Do we need this comment all the time or can we just rename the function into setupTwoTables? :)